File: QE-Selenium-Java/NavComAct3.java

package Demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class NavComAct3 {

    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://training-support.net/webelements/login-form");
        
        System.out.println(driver.getTitle() + " before login");

        WebElement ele1 = driver.findElement(By.xpath("//input[@id='username']"));
        ele1.sendKeys("admin");

        WebElement ele2 = driver.findElement(By.xpath("//input[@id='password']"));
        ele2.sendKeys("password");

        WebElement ele3 = driver.findElement(By.xpath("//button[@class='svelte-1pdjkmx']"));
        ele3.click();

        System.out.println(driver.getTitle() + " after login");

        driver.quit();
    }
}

File: QE-Selenium-Python/NavComAct3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form")
    
    print(driver.title)
    
    ele1 =driver.find_element(By.XPATH,"//input[@id='username']")
    ele1.send_keys("admin")
    
    ele2 =driver.find_element(By.XPATH,"//input[@id='password']")
    
    ele2.send_keys("password")
    
    driver.find_element(By.XPATH,"//button[@class='svelte-1pdjkmx']").click()
    
    print(driver.title)
    
    driver.quit()
    
        
    